### PR TITLE
Dockerfile Created

### PR DESCRIPTION
**Pull request by xShuden**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,20 @@ git clone https://github.com/mvelazc0/PurpleSpray.git
...
+ ### Docker Build
+
+ ```
+ $ docker build -t xshuden/purplespray .
+ ```
+
+### Docker Usage
+
+ ```
+ $ docker run --rm -it xshuden/purplespray
```
Comment (mvelazc0) [inline README.md:27]: Thanks for the PR !!

Does this command need the "xshuden"  ?

Comment (xShuden) [reply README.md:27]: No not. It can be build with any name.

Comment (xShuden) [reply README.md:27]: Build example: docker build -t purplespray .
Run example: docker run --rm -it purplespray

Comment (mvelazc0) [reply README.md:27]: Thanks. I will change that part, test docker and merge the PR.

Comment (xShuden) [reply README.md:27]: Thanks.  You can test it here. https://hub.docker.com/r/xshuden/purplespray

**Review by mvelazc0 — APPROVED** [review]
Tested creating the container from scratch.

**Comment by mvelazc0** [comment]
Merged. Thanks again !

**Pull request merged**